pkg/kubeapiserver/admission/initializer.go

```diff
@@ -32,40 +29,15 @@ type WantsCloudConfig interface {
 
 // PluginInitializer is used for initialization of the Kubernetes specific admission plugins.
 type PluginInitializer struct {
-	cloudConfig                []byte
-	quotaConfiguration         quota.Configuration
-	excludedAdmissionResources []schema.GroupResource
+	CloudConfig []byte
```

Comment:
Note to the reviewer: everything but the cloud config is moved into pkg/controlplane/apiserver/admission/config.go